fastlane/lib/fastlane/actions/clear_derived_data.rb

```diff
@@ -17,15 +28,15 @@ def self.description
       end
 
       def self.details
-        "Deletes the Derived Data from '~/Library/Developer/Xcode/DerivedData' or a supplied path"
+        "Deletes the Derived Data from path set on Xcode or a supplied path"
       end
 
       def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :derived_data_path,
                                        env_name: "DERIVED_DATA_PATH",
                                        description: "Custom path for derivedData",
-                                       default_value: "~/Library/Developer/Xcode/DerivedData")
+                                       default_value: xcode_preferences['IDECustomDerivedDataLocation'] || "~/Library/Developer/Xcode/DerivedData")
```

Comment:
Can you add `default_value_dynamic: true` into the initializer of this `ConfigItem`? Its a way to note that the default value is calculated dynamically